src/components/MainPage/ReportProblemDialog.vue

```diff
@@ -0,0 +1,158 @@
+<template>
+  <v-dialog v-model="dialog">
+    <v-card>
+      <v-card-title>
+        <span class="headline">Report a problem</span>
+      </v-card-title>
+      <v-card-text>
+        <v-form
+          ref="form"
+          v-model="valid"
+        >
+          <v-container>
+            <v-row>
+              <v-col>
+                <v-text-field
+                  v-model="formData.title"
+                  label="Title"
+                  hint="Describe the bug in one sentence"
+                  persistent-hint
+                  required
+                  :rules="[titleRule]"
+                />
+              </v-col>
+            </v-row>
+            <v-row>
+              <v-col>
+                <v-textarea
+                  v-model="formData.detailes"
+                  label="Bug Details"
+                  filled
+                  :rules="[detailesRule]"
+                  :required="!formData.attachLogs"
+                />
+              </v-col>
+            </v-row>
+            <v-row>
+              <v-col>
+                <v-checkbox
+                  v-model="formData.attachLogs"
+                  :required="!formData.detailes"
+                >
+                  <template slot="label">
+                    Attach logs (<a @click.prevent="sheet = true">see</a>)
+                    <log-sheet
+                      v-model="sheet"
+                      :raw="raw"
+                    />
+                  </template>
+                </v-checkbox>
+              </v-col>
+            </v-row>
+          </v-container>
+          <small>*indicates required field</small>
+        </v-form>
+      </v-card-text>
+      <v-card-actions>
+        <v-btn
+          color="blue darken-1"
+          text
+          @click="dialog = false"
+        >
+          Close
+        </v-btn>
+        <v-spacer />
+        <v-btn
+          color="blue darken-1"
+          text
+          :disabled="!valid"
+          @click="open"
+        >
+          Open Github Issue
+        </v-btn>
+      </v-card-actions>
+    </v-card>
+  </v-dialog>
+</template>
+
+<script>
+import { shell } from 'electron';
+import LogSheet from './LogSheet';
+
+const createGithubIssueLink = (title, detailes, log) => {
+  const formattedDetailes = detailes ? `
+## Detailes
+
+${detailes}` : '';
+
+  const formattedLog = log ? `
+## Log
+\`\`\`
+${log}
+\`\`\`` : '';
+
+  return 'https://github.com/baruchiro/israeli-bank-scrapers-desktop/issues/new?'
+          + `title=${escape(title)}`
+          + `&body=${escape(formattedDetailes)}${escape(formattedLog)}`;
```

Comment:
```suggestion
          + `title=${encodeURI(title)}`
          + `&body=${encodeURIComponent(formattedDetailes)}${encodeURIComponent(formattedLog)}`;
```

(just for the record, the escape function is deprecated)